validate.go

```diff
@@ -5,22 +5,25 @@ import "fmt"
 // ValidateService handles communication with the validation related methods of
 // the GitLab API.
 //
-// GitLab API docs: https://docs.gitlab.com/ce/api/lint.html
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/lint.html
```

Comment:
Thanks for trying to make things consistent! Yet they do not all need to be formatted like this. It's bases on line length. The target for doc lines is to keep it below 80 chars... Of course a URL can still be longer, but we should not break in the middle of a URL so that is fine as long as these longer URLs start on their own line.

Yet this one is short enough, so this can just be placed on one line.

Reply:
Ha-ha :) Yes, I understand. Okay, I'll fix it.